WinImageBuilder.psm1

```diff
@@ -616,6 +615,33 @@ function Compress-Image {
     Write-Output "MaaS image is ready and available at: $ImagePath"
 }
 
+function Create-ProtectedZip {

```

Comment:
Leave the method name unchanged Start-WindowsCommand and use the alias param when you use it.